app/views/orders/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="row">
+  <h1 class="header-box"> Order <%= @order.id %></h1>
+  <h5 class="cntr-header">Order status: <%= @order.status.capitalize %></h5>
+</div>
+<div class="row">
+  <div class="col-xs-12">
+    <h2>
+      Customer Information
+    </h2>
+    <strong>Name: <%= @order.cc_name %></strong><br>
+    <strong>Email:</strong> <%= @order.email_address %><br>
+    <strong>Mailing Address: </strong><%= @order.mailing_address %><br>
```

Comment:
Same with the <strong> tags, it's better practice to add this styling in the css, instead of hard coding it in the html